script/start-gke-env.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+CLUSTER=${1:?}
+ZONE=${2:?}
+VERSION=${3:?}
+ADMIN=${4:?}
+
+function clean() {
+    local resource=${1:?}
+    kubectl get $resource | awk '{print $1}' | xargs kubectl delete $resource || true
+}
+if ! gcloud container clusters list; then
+    echo "Unable to access gcloud project"
+    exit 1
+fi
+
+if gcloud container clusters list | grep -q $CLUSTER; then
+    echo "GKE cluster already exits. Deleting resources"
+    # Cluster already exists, make sure it is clean
+    gcloud container clusters get-credentials $CLUSTER --zone $ZONE
+    kubectl delete ns kubeless || true
+    resources=(
+        functions
+        cronjobs
+        jobs
+        deployments
+        horizontalpodautoscalers
+    )
+    for res in "${resources[@]}"; do
+        clean $res
+    done
+    kubectl delete crd functions.k8s.io || true
+else
+    echo "Creating cluster $CLUSTER in $ZONE (v$VERSION)"
+    # Bypass the warning about using alpha features
+    echo 'y' | gcloud container clusters create --cluster-version=$VERSION --zone $ZONE $CLUSTER --num-nodes 5 --no-enable-legacy-authorization --enable-kubernetes-alpha
```

Comment:
what alpha features do we need?

Reply:
CronJobs (for scheduled functions) and HorizontalPodAutoscalers (for autoscaling functions) are alpha in Kubernetes 1.7

Reply:
:+1: